datajunction-server/datajunction_server/models/user.py

```diff
@@ -0,0 +1,40 @@
+"""
+Models for users and auth
+"""
+from enum import Enum
+from typing import Optional
+
+from pydantic import BaseModel
+from sqlmodel import Field, SQLModel
+
+
+class OAuthProvider(Enum):
+    """
+    Support oauth providers
+    """
+
+    BASIC = "basic"
+    GITHUB = "github"
+
+
+class User(SQLModel, table=True):  # type: ignore
+    """Class for a user."""
+
+    id: Optional[int] = Field(default=None, primary_key=True)
+    username: str
+    password: Optional[str]
+    email: Optional[str]
+    name: Optional[str]
+    oauth_provider: OAuthProvider
+    is_admin: bool = False
+
+
+class UserOutput(BaseModel):
```

Comment:
Not related to this PR, but I was wondering if adding an `Output` suffix is a common way to create a simplified class for returning it in the HTTP requests?